index.html

```diff
@@ -0,0 +1,386 @@
+</div>
+
+### Fold
+
+<dfn>Fold</dfn> is used as both a verb and a noun in the context of this explainer. _To fold_ means to lay one [part of the] screen over another [part of the] screen. _A fold_ (aka a screen fold) as a noun refers to the flexible [=screen area=] that bends.
+
+### CSS pixel
+
+A <dfn>CSS pixel</dfn> is denoted in CSS with the suffix px. It is a unit of length which roughly corresponds to the width or height of a single dot that can be comfortably seen by the human eye without strain ([MDN](https://developer.mozilla.org/en-US/docs/Glossary/CSS_pixel)).
+
+One CSS pixel doesn’t necessarily correspond to one physical screen pixel, since that would make everything very small on high resolution screens. Instead one such CSS pixel might correspond to for example 2x2 square of physical pixels - or even non-discrete values given a subpixel (red, green, and blue) layout.
+
+Many OS’s allow to change the scaling of all UI elements which often is reflected in the [device pixel ratio](https://drafts.csswg.org/cssom-view/#dom-window-devicepixelratio) that makes up one CSS pixel. A browser also often allows the user to apply a scale on top of the OS-provided scale on a per site basis. The layout is laid out considering the browser viewport in CSS pixels. For example, given a browser scale of 125%, a 800px wide viewport becomes a 640px wide viewport.
+
+### Layout zoom
+
+<div class="issue">
+<dfn>Layout zoom</dfn> to be defined.
+</div>
+
+### Content scaling
+
+Usually content is laid out given the size of the viewport in [=CSS pixels=], but that may not always be the case. Especially on mobile devices the user may select “Show desktop site” which will usually lay the content out mimicking common desktop browser window size (typically 980-1024px width), and then that content is scaled so that the width fits within the actual viewport.
+
+Typically you can control this scaling via the viewport meta tag and browsers often also allow the user to manually change the content scale, by so-called pinch zoom gestures.
+
+### Window manager
+
+<div class="issue">
+<dfn>Window manager</dfn> to be defined. In particular, articulate the constraints imposed by various window managers (Android, split-screen, desktop/tablet modes, Windows 10X etc.)
+</div>
+
+### Content area bounds
+
+<div class="issue">
+<dfn>Content area bounds</dfn> to be defined.
+</div>
+
+## What are we trying to solve?
+
+For [=foldable devices=]:
+- Let developers design around the hinge for [=dual-screen devices=] when in [=spanning=] mode
+- React to [=posture=] changes to allow the possibility to re-layout accordingly
+- Access the hinge size, type and angle
+
+For [=multi-monitor setups=]:
+- Ability to layout and display a window on any of the connected screens
```

Comment:
This first and third points seem like duplicates; I might change the points to:
For [=multi-monitor setups=], let web applications:
- Request information about all screens in the [=overall visual workspace=]
- Request fullscreen on any screen in the [=overall visual workspace=]
- Request placement of non-fullscreen windows on any screen in the [=overall visual workspace=]

Reply:
Addressed by 13de491.